src/plugins/ciu/ciuPlugin.js

```diff
@@ -0,0 +1,73 @@
+const db = require('caniuse-db/data.json');
+const { messageToFactoid } = require('../factoids/factoidsPlugin');
+
+const featureKeys = Object.keys(db.data);
+
+const AGENTS_TO_SHOW = {
+  ie: 'IE',
+  edge: 'Edge',
+  firefox: 'FF',
+  chrome: 'Chrome',
+  opera: 'Opera',
+  safari: 'Safari',
+  ios_saf: 'iOS',
+  android: 'Android',
+};
+
+function getAgentSupportArray(feature) {
+  return Object.entries(AGENTS_TO_SHOW).flatMap(([agent, agentName]) => {
+    let supportedVersions = db.agents[agent].versions.filter(
+      (v) => v && !/[nupax]/g.test(feature.stats[agent][v]),
+    );
+
+    if (supportedVersions.length === 0) {
+      supportedVersions = db.agents[agent].versions.filter(
+        (v) => v && !/[nu]/g.test(feature.stats[agent][v]),
+      );
+    }
+
+    return supportedVersions.length > 0
+      ? [`${agentName}${supportedVersions[0]}`]
+      : [];
+  });
+}
+
+module.exports = async function ciuPlugin(msg) {
+  if (!msg.command) return;
+
+  const [cmd, ...search] = msg.command.command.split(' ');
+  if (cmd !== 'ciu' || search.length === 0) {
+    return;
+  }
```

Comment:
should maybe `!ciu` error out? or is that ok to be a factoid

Reply:
dunno, if we don't have a useful error, better none I guess